### PR TITLE
chore: Prepare 8.5.1

### PR DESCRIPTION
**Pull request by susnux**
## [v8.5.1](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.5.1) (2024-01-24)
[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v8.5.0...v8.5.1)

### 🐛 Fixed bugs
* fix(NcPopover): a11y attrs provide-inject reactivity [\#5142](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5142) \([ShGKme](https://github.com/ShGKme)\)
* fix(NcCheckboxRadioSwitch): On button style align the text vertically [\#5151](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5151) \([susnux](https://github.com/susnux)\)
* fix: removed groupings in timezone picker [\#5130](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5130) \([emoral435](https://github.com/emoral435)\)
* fix(fields): adjust label line-height [\#5133](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5133) \([ShGKme](https://github.com/ShGKme)\)
* fix(header): width regression fixed [\#5129](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5129) \([emoral435](https://github.com/emoral435)\)
* fix(NcCheckboxRadioSwitch): Make sure to include current value in cheched array [\#5150](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5150) \([susnux](https://github.com/susnux)\)
* fix: Ensure no double scrollbars for NcDialog and NcAppNavigationSettings [\#5139](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5139) \([susnux](https://github.com/susnux)\)

### Other Changes
* Updated translation
* chore: Fix missing PR from changelog [\#5125](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5125) \([Pytal](https://github.com/Pytal)\)
* chore: Migrate cypress config to vite instead of webpack [\#4961](https://github.com/nextcloud-libraries/nextcloud-vue/pull/4961) \([susnux](https://github.com/susnux)\)
* chore(styleguide): set line-height from server styles [\#5132](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5132) \([ShGKme](https://github.com/ShGKme)\)

### CODE REVIEW

**Review by ShGKme — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Review by Antreesy — COMMENTED** [review]
Last chance to remind that we're one step away from fixin user mentions in Talk and Files: 
* https://github.com/nextcloud-libraries/nextcloud-vue/pull/5138

**Comment by Pytal** [comment]
Re-review here https://github.com/nextcloud-libraries/nextcloud-vue/pull/5131 appreciated!

**Comment by susnux** [comment]
We can easily have a 8.5.2 or even 8.6.0 next week, we just need a bug fix for the release urgent.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,26 @@
...
+[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v8.5.0...v8.5.1)
+
+### 🐛 Fixed bugs
+* fix(NcPopover): a11y attrs provide-inject reactivity [\#5142](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5142) \([ShGKme](https://github.com/ShGKme)\)
+* fix(NcCheckboxRadioSwitch): On button style align the text vertically [\#5151](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5151) \([susnux](https://github.com/susnux)\)
+* fix: removed groupings in timezone picker [\#5130](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5130) \([emoral435](https://github.com/emoral435)\)
+* fix(fields): adjust label line-height [\#5133](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5133) \([ShGKme](https://github.com/ShGKme)\)
+* fix(header): width regression fixed [\#5129](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5129) \([emoral435](https://github.com/emoral435)\)
+* fix(NcCheckboxRadioSwitch): Make sure to include current value in cheched array [\#5150](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5150) \([susnux](https://github.com/susnux)\)
```
Comment (Antreesy) [inline CHANGELOG.md:14]:
```suggestion
* fix(NcCheckboxRadioSwitch): Make sure to include current value in checked array [\#5150](https://github.com/nextcloud-libraries/nextcloud-vue/pull/5150) \([susnux](https://github.com/susnux)\)
```

**Pull request merged**